app/models/court_application.rb

```diff
@@ -3,39 +3,33 @@
 class CourtApplication < ApplicationRecord
   include BuilderMappable
   include CourtCentreRelatable
-  belongs_to :application_type, class_name: "CourtApplicationType"
+
   belongs_to :court_application_party
-  belongs_to :court_application_outcome, optional: true
   belongs_to :court_application_payment, optional: true
-  has_many :court_application_respondents
+  belongs_to :court_application_type
+
   has_many :judicial_results
 
-  validates :application_type, presence: true
   validates :applicationReceivedDate, presence: true
   validates :court_application_party, presence: true
   validates :applicationStatus, presence: true, inclusion: %w[DRAFT LISTED FINALISED]
 
   def to_builder
     Jbuilder.new do |court_application|
       court_application.id id
-      court_application.type application_type.to_builder
       court_application.applicationReceivedDate applicationReceivedDate.to_date
       court_application.applicationReference applicationReference
       court_application.applicant court_application_party.to_builder
-      court_application.respondents array_builder(court_application_respondents)
-      court_application.applicationOutcome court_application_outcome.to_builder if court_application_outcome.present?
-      court_application.linkedCaseId linkedCaseId
-      court_application.linkedSplitProsecutorCaseReference linkedSplitProsecutorCaseReference
+      court_application.respondents
```

Comment:
`court_application.respondents array_builder(court_application_party)` ?

Reply:
good spot! I've removed this field as its not required by the schema. If needed, it can be added back in the future

Reply:
ok no worries